### ISSUE TITLE
Unmet peer dependency warning with unocss@0.56.5 and @ikun-ui/core@0.0.9

### ISSUE THREAD

**Issue by cat394** [ikun-svelte/ikun-ui#292]
### Description

Hello,

I recently encountered a warning regarding an unmet peer dependency when using `@ikun-ui/core` version `0.0.9` alongside `unocss` version `0.56.5`.

Here is the warning message I received:

```bash
devDependencies:
+ unocss 0.56.5

 WARN  Issues with peer dependencies found
.
└─┬ @ikun-ui/core 0.0.9
  └── ✕ unmet peer unocss@^0.54.1: found 0.56.5
```
​
Based on the version constraint `^0.54.1` specified in `peerDependencies`, I believe `unocss@0.56.5` should be considered compatible. I'm not sure if this is an issue with the dependency specification in `@ikun-ui/core` or if there's something else I might be missing.

Any guidance on this matter would be appreciated. Thank you for your time and assistance.

**Comment by baiwusanyu-c** [comment]
It should not affect usage. `peerDependencies` will be updated in the next version.

**Comment by cat394** [comment]
Thank you for your reply. I got it.

**Comment by baiwusanyu-c** [comment]
fix wit 8f65178c3b0ad11e1536abe2e8aa04c1e334b75a
